python/runtime/cudaq/builder/py_kernel_builder.cpp

```diff
@@ -347,6 +347,22 @@ void bindKernel(py::module &mod) {
       ADD_BUILDER_PARAM_QIS_METHOD(r1)
       // clang-format on
 
+      /// @brief Bind the SWAP gate.
+      .def(
+          "swap",
+          [](kernel_builder<> &self, const QuakeValue &first,
+             const QuakeValue &second) { return self.swap(first, second); },
+          py::arg("first"), py::arg("second"),
+          "Swap the states of the provided qubits.\n "
```

Comment:
Just curious ... but doesn't python support that triple string syntax? `"""` ... `"""`.

Reply:
I've had a nightmare with sphinx trying to build off of R-strings. I'm going to file a refactor ticket for this so I can properly sort it out 😄

Reply:
Oh. Well, it seemed like a good idea at the time. lol